pallets/schemas/src/lib.rs

```diff
@@ -306,20 +324,54 @@ pub mod pallet {
 		/// * [`Error::InvalidSchema`] - Schema is malformed in some way
 		/// * [`Error::SchemaCountOverflow`] - The schema count has exceeded its bounds
 		#[pallet::call_index(3)]
-		#[pallet::weight(T::WeightInfo::create_schema_via_governance(model.len() as u32))]
+		#[pallet::weight(T::WeightInfo::create_schema_via_governance(model.len() as u32+ settings.len() as u32))]
 		pub fn create_schema_via_governance(
 			origin: OriginFor<T>,
 			creator_key: T::AccountId,
 			model: BoundedVec<u8, T::SchemaModelMaxBytesBoundedVecLimit>,
 			model_type: ModelType,
 			payload_location: PayloadLocation,
+			settings: BoundedVec<SchemaSetting, T::MaxSchemaSettingsPerSchema>,
```

Comment:
@shannonwells  this is also updated, if this is not a breaking change, we can drop the extrinsic below and keep this

Reply:
I think since this is only get triggered by governance, it should not not be considered breaking.